recipes/radiospectra/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "radiospectra" %}
+{% set version = "0.1" %}
+{% set sha256 = "b38651c1bb9ea2f0b720ed16faa2da31cebff07d53779edf39fae4cb0fa43bb4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sunpy >=0.8
+
+  run:
+    - python
+    - sunpy >=0.8
+    - astropy >=2.0
+    - numpy
+    - scipy
+    - matplotlib
+
+test:
+  imports:
+    - radiospectra
+    - radiospectra.sources
+    - radiospectra.tests
+about:
+  home: http://sunpy.org
+  license: BSD-3-Clause
```

Comment:
Needs a license_family and a license_file

Reply:
I've got troubles because the license_file is a `rst`. I'm changing that on pypi first.